week1/main.css

```diff
@@ -0,0 +1,179 @@
+  box-shadow: 15px 15px 15px #888888;
+  text-decoration: none;
+}
+
+/*Add-head-logo styling end*/
+
+/* Brand-logo styling*/
+.brand-logo img{
+  border-radius: 50%;
+  border: 5px solid #59baf2;
+  padding: 0px;
+  border-color: #4a4a4a;
+  margin-left: auto;
+  margin-right: auto
+}
+/* Brand-logo styling finished*/
+
+/*Navigation bar styling*/
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #505045;
+  border-radius: 5px;
+  border-style:hidden;
+  box-shadow: 15px 15px 15px #888888;
+  text-decoration: none;
+  
+}
+
+li {
+  float: left;
+}
+
+li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  font-size: 30px;
+  text-shadow: 5px 5px 10px blue"
+}
+
+li a:hover:not(.active) {
+  background-color: #101111;
+}
+
+.active {
+  background-color: #925129;
+}
+a:link, a:visited{
+  text-decoration:none;
+}
+/*Navigation bar end*/
+
+/* Blog-main styling*/
+.blog-main{
+  border-style:hidden;
+  border-radius: 8px;
+  border-width: 3px;
+  border-color:#101111;
+  padding: 20px;
+  margin: 5px;
+  background-color: #f5fdce;
+  box-shadow: 5px 10px 10px #888888;
+}
+
+h1{
+  text-decoration: underline;
+  padding: 20px;
+  margin: 5px;    
+  text-align: center;
+}
+/* Blog-main styling finished*/
+
+/* Reference P styling*/
+.blog-main p:last-of-type{
+  font-size: 15px;
+  font-family: sans-serif;
+  font-weight: bolder;
+  font-style: italic;
+  text-align: right;
+}
+/* Reference P styling end*/
+
+/* Articles-section styling*/
+section{
+  display: block;
+  justify-content: center;
+} 
+
+article{
+  border-style:hidden;
+  font-family: 'Times New Roman';
+  border-width: 5px;
+  border-radius: 8px;
+  border-color:#101111;
+  padding: 20px;
+  margin: 5px;
+  background-color: #fde1fd;
+  box-shadow: 5px 15px 15px #464545;
+}
+
+article p:last-of-type{
+  font-size: 20px;
+  font-family:'Times New Roman';
+  border-style:hidden;
+  border-width: 5px;
+  border-radius: 8px;
+  margin: 0px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 10px;
+  border-color:#101111;
+  background-color: #f5eff5;
+  box-shadow: 5px 15px 15px #464545;
+  text-align: center;
+}
+article p:first-of-type{
+  font-size: 30px;
+  font-family:'Times New Roman';
+  font-style: italic;
+}
+/* Articles-section styling end*/
+
+/* Footer styling*/
+footer{
+  border-style:hidden;
+  border-width: 3px;
+  border-radius: 8px;
+  border-color:#101111;
+  padding: 20px;
+  margin: 5px;
+  background-color: #a2a4ac;
+  box-shadow: 5px 10px 8px #888888;
+  font-family: 'Lucida Sans';
+}
+
+footer p:last-of-type{
+  text-align: center;
+  font-size: 30px;
+  font-weight: bolder;
+  font-style: italic;
+}
+table{
+  font-size: 30px;
+  text-decoration: underline;
+  border-spacing: 20px;
+  justify-content: space-around;
+  width: 100%;
+}
+table a{
+  color:black;
+}
+table a:hover{
+  color:blue;
+}
+/* Footer styling*/
+/*end of the code
```

Comment:
Always close your comments; if this were anywhere else it would cause a *lot* of problems

Reply:
Done.